packages/@o3r/localization/src/devkit/localization-devtools.contextualization.service.ts

```diff
@@ -0,0 +1,99 @@
+import { ApplicationRef, Inject, Injectable, Optional } from '@angular/core';
+import type { DevtoolsServiceInterface, WindowWithContextualization } from '@o3r/core';
+import { LocalizationService } from '../tools';
+import { LocalizationDevtoolsServiceOptions } from './localization-devkit.interface';
+import { OTTER_LOCALIZATION_CONTEXTUALIZATION_DEFAULT_OPTIONS, OTTER_LOCALIZATION_CONTEXTUALIZATION_OPTIONS } from './localization-devtools.token';
+import { lastValueFrom, Subscription, take } from 'rxjs';
+
+@Injectable()
+export class LocalizationDevtoolsContextualizationService implements DevtoolsServiceInterface {
```

Comment:
Why creating a new class instead of adding the function in the `LocalizationDevtoolsConsoleService` (just need to create an interface to ensure having the function required by the editor)?

Reply:
I don't want both class to be instanciate and activated with the same condition

Reply:
But they give access to the same level of custo (via the same entry) and it will just force the app dev to import 2 devkit to activate instead of 1